wdpk/docker/install.sh

```diff
@@ -77,8 +79,8 @@ sleep 1
 
 sleep 3
 
-# install portainer to manage docker
-docker ps | grep portainer
+# install portainer to manage docker, only if there is container Portainer (running or not)
```

Comment:
So this will only install portainer if it was never installed before? That makes sense I guess.

Reply:
There is a typo issue in the comment. 
.. there is no portainer container..